internal/provider/data_source_packer_image.go

```diff
@@ -132,5 +134,9 @@ func dataSourcePackerImageRead(ctx context.Context, d *schema.ResourceData, meta
 		}
 	}
 
+	if !found {
+		return diag.Errorf("Unable to load image with region %s and cloud %s for iteration %s.", region, cloudProvider, iterationID)
```

Comment:
Should this error change if there's no cloud provider passed?

Reply:
see above; the cloud provider must always be passed. It was a docs mistake.